src/sql/workbench/services/accountManagement/test/browser/accountManagementService.test.ts

```diff
@@ -573,13 +573,13 @@ function getFailingMockAccountProvider(cancel: boolean): TypeMoq.Mock<azdata.Acc
 	mockProvider.setup(x => x.prompt())
 		.returns(() => {
 			return cancel
-				? Promise.resolve(<azdata.PromptFailedResult>{ canceled: true }).then()
+				? Promise.resolve(<azdata.ProviderError>{ canceled: true }).then()
```

Comment:
Given that you're adding in the canceled property then these should still be PromptFailedResult, right?

But also you shouldn't need the casting I would think - the type it's expecting should be a ProviderError already so it should be ok with it based on the properties of the object being returned.